Section 3 - Base Features & Syntax/End/src/Person/Person.js

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
 
 const person = (props) => {
-    return <p>I'm {props.name} and I am {props.age} years old!</p>;
+  return (
+    <div>
+      <p>I&apos;m {props.name} and I am {props.age} years old!</p>
+      <p>{props.children}</p>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['children' is missing in props validation](https://app.codacy.com/manual/saylormic/react-ultimate-guide/pullRequest?prid=4954074)